salt/kubelet/kubelet-config.jinja

```diff
@@ -0,0 +1,61 @@
+kind: KubeletConfiguration
+apiVersion: kubelet.config.k8s.io/v1beta1
+{%- if grains['lsb_distrib_id'] == "CAASP" %}
+staticPodPath: "/etc/kubernetes/manifests"
+{%- endif %}
+syncFrequency: 1m0s
+fileCheckFrequency: 20s
+httpCheckFrequency: 20s
+address: 0.0.0.0
+port: {{ pillar['kubelet']['port'] }}
+tlsCertFile: "/var/lib/kubelet/pki/kubelet.crt"
+tlsPrivateKeyFile: "/var/lib/kubelet/pki/kubelet.key"
```

Comment:
The `kubelet.{crt|key}` are specified [here in the pillar](https://github.com/kubic-project/salt/blob/master/pillar/certificates.sls#L53)

Reply:
I have changed this